pkg/serverless/serverless.go

```diff
@@ -172,9 +172,6 @@ func WaitForNextInvocation(stopCh chan struct{}, daemon *daemon.Daemon, id regis
 			metricTags := tags.AddColdStartTag(daemon.ExtraTags.Tags, daemon.ExecutionContext.Coldstart)
 			metricsChan := daemon.MetricAgent.GetMetricChannel()
 			metrics.SendTimeoutEnhancedMetric(metricTags, metricsChan)
-			if err != nil {
```

Comment:
this is unrelated to this PR but this useless log should have been removed in v10 refactoring